SofaKernel/modules/SofaBaseVisual/BaseCamera.cpp

```diff
@@ -834,6 +930,43 @@ void BaseCamera::draw(const sofa::core::visual::VisualParams* /*params*/)
 }
 
 
+void BaseCamera::drawCamera(const core::visual::VisualParams* vparams)
+{
+    glEnable(GL_MULTISAMPLE_ARB);
```

Comment:
1- why multisampling?
2- mix of drawtool and opengl...

Reply:
Yes, you are totally right

sorry I marked it as to review too early. I will fix all that.

Reply:
It is fixed by I'm now facing an issue on the windows build, I'm not sure it is related with the PR.